patches/minecraft/net/minecraft/advancements/PlayerAdvancements.java.patch

```diff
@@ -0,0 +1,33 @@
+--- ../src-base/minecraft/net/minecraft/advancements/PlayerAdvancements.java
++++ ../src-work/minecraft/net/minecraft/advancements/PlayerAdvancements.java
+@@ -27,6 +27,8 @@
+ import net.minecraft.util.JsonUtils;
+ import net.minecraft.util.ResourceLocation;
+ import net.minecraft.util.text.TextComponentTranslation;
++import net.minecraftforge.common.MinecraftForge;
++import net.minecraftforge.event.entity.player.AdvancementEvent;
+ import org.apache.logging.log4j.LogManager;
+ import org.apache.logging.log4j.Logger;
+ 
+@@ -208,12 +210,21 @@
+ 
+             if (!flag1 && advancementprogress.func_192105_a())
+             {
++                if (MinecraftForge.EVENT_BUS.post(new AdvancementEvent.Pre(this.field_192762_j, p_192750_1_))) {
```

Comment:
...change this line to `net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.event.entity.player.AdvancementEvent.Pre(this.field_192762_j, p_192750_1_))) {`, ...